zerver/openapi/zulip.yaml

```diff
@@ -5895,6 +5895,51 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/JsonSuccess"
+    patch:
+      operationId: update_linkifier
+      tags: ["server_and_organizations"]
+      description: |
+        Update [linkifiers](/help/add-a-custom-linkification-filter), regular
+        expression patterns that are automatically linkified when they appear
+        in messages and topics.
+
+        `PATCH {{ api_url }}/v1/realm/filters/{filter_id}`
+      parameters:
+        - name: filter_id
+          in: path
+          description: |
+            The ID of the linkifier that you want to update.
+          schema:
+            type: integer
+          example: 42
+          required: true
+        - name: pattern
+          in: query
+          description: |
+            The [Python regular
+            expression](https://docs.python.org/3/howto/regex.html) that should
+            trigger the linkifier.
+          schema:
+            type: string
+          example: "#(?P<id>[0-9]+)"
+          required: true
+        - name: url_format_string
+          in: query
+          description: |
+            The URL used for the link. If you used named groups for the `pattern`,
+            you can insert their content here with
+            `%(name_of_the_capturing_group)s`.
+          schema:
+            type: string
+          example: https://github.com/zulip/zulip/issues/%(id)s
+          required: true
```

Comment:
For these last two parameters, since they should match what's present with the "add realm filter" endpoint (and I assume you just copied), it's worth extracting this as a shared variable.

Reply:
Sorry, I didn't get what `extracting this as a shared variable` means here.

Reply:
We have a set of common components in the API docs like `#/components/schemas/BasicStream` that let us avoid duplicating code in the OpenAPI files -- we should use that mechanism for this as well.